src/webui/www/private/scripts/prop-webseeds.js

```diff
@@ -1,3 +1,31 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2015  Diego de las Heras <ngosang@hotmail.es>
+ *
```

Comment:
Commit 8e4b9f30bb72a213fc840aa755481ce9e5a908be shows as name/email `ngosang <diegodelasheras@gmail.com>`
@ngosang which one do you want to be put here?

Reply:
In case he is too busy, he used the hotmail one in a recent commit: https://github.com/qbittorrent/search-plugins/commit/b1e9b5c91337b2cd6d1072a08b6161edb16d3bdf.patch

Reply:
@sledgehammer999 @Chocobo1 `ngosang@hotmail.es` is the right one.  @sledgehammer999 please, edit your comment to remove my gmail, i don't want spam there...